src/components/WebexMeetingGuestAuthentication/WebexMeetingGuestAuthentication.jsx

```diff
@@ -58,11 +65,26 @@ export default function WebexMeetingGuestAuthentication({
     phone: width <= PHONE_LARGE,
   });
 
+  let passwordError = '';
+  let captchaError = '';
+
+  if (invalidPassword && failureReason === 'WRONG_PASSWORD') {
+    if (!requiredCaptcha.captchaId) {
+      passwordError = 'Incorrect password. Try again.';
+    } else {
+      captchaError = 'Incorrect password entered too many times. Enter captcha code';
```

Comment:
Do you think it is okay to put these in a constant and use it from there? This will help us in the longer run to start supporting other languages in widgets

Reply:
Creating a JIRA ticket to move all the translations and constants to a separate file. 
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-538547

Reply:
As an extension of this, can we create a ticket for adding ReactIntl as well? Looks like all the messages are in English. We should add other languages for accessibility.

Reply:
I think the above ticket also aims the same. We can combine both of them right?

@Shreyas281299 @sreenara

Reply:
Created a JIRA to use reactIntl to render text - https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-539304